includes/Core/Util/Uninstallation.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Util\Uninstallation
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Core\Storage\Encrypted_Options;
+use Google\Site_Kit\Core\Authentication\Credentials;
+use Google\Site_Kit\Core\Authentication\Google_Proxy;
+use Exception;
+
+/**
+ * Utility class for handling uninstallation of the plugin.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Uninstallation {
+
+	/**
+	 * Plugin context.
+	 *
+	 * @since n.e.x.t
+	 * @var Context
+	 */
+	private $context;
+
+	/**
+	 * Options object.
```

Comment:
Super nitpick but this is how it's defined most places.
```suggestion
	 * Options instance.
```